transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Service/Tree.java

```diff
@@ -82,61 +81,66 @@ public Node hasChild(String intent){
             return null;
         }
 
```

Comment:
Good job on adding helper functions to the code rather than having a block of code